test/index.coffee

```diff
@@ -169,13 +169,13 @@ describe 'Document', ->
     cursor = MyDoc.collection.find {}
     cursor.limit 1, (err, docs) ->
       throw err if err
-      docs.should.have.length.of 1
+      docs.should.have.lengthOf 1
```

Comment:
It seems like Chai syntax has changed in later versions. Updated to fix tests after upgrading chai.